ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/interface_groups.schema.yml

```
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  interface_groups:
+    type: list
+    primary_key: name
```

This key is a bit special, since it is part of the maintenance feature, so I suggest a special display name:
```suggestion
    display_name: Maintenance Interface Groups
    primary_key: name
```